textDisplay.cc

```diff
@@ -0,0 +1,82 @@
+#include "textdisplay.h"
+
+TextDisplay::TextDisplay(int n):View(n){
+	// Creates a new nxn textdisplay and sets the default value to '0'
+	theDisplay = new char*[n];
+	for (int row = 0; row < n; row++){
```

Comment:
Right, when we initialize the game or run "clearGame()" to reset the board, we'll assign the pieces to each cell and then the cells will notify the view.